test/test.py

```diff
@@ -5,6 +5,20 @@
 from nose.plugins.attrib import attr
 
 
+def getDelta(num):
+    if num == None:
+        return 0
+    if num <= 500:
+        return 50
+    if num <= 1000:
+        return 100
+    if num <= 2000:
+        return 200
+    if num <= 4000:
+        return 400
+    return 500
```

Comment:
This is basically computing 10% tolerance, which could be simplified:
```suggestion

    return num * .1
```

Reply:
Sure, but I guess we don't want with expected value 5  delta 0.5 or with 60.000 6.000? At some point it should be constant.

Reply:
Hardcoded 500 hardly solves anything as we have negative delta 600+ again in February with 40k vaccinations...

Reply:
I surrender :)